frontend/model/database.js

```diff
@@ -164,19 +164,70 @@ const files = localforage.createInstance({
   storeName: 'Files'
 })
 
+const maxFileEntries = 100
+
 sbp('sbp/selectors/register', {
-  'gi.db/files/save': function (url: string, blob: Blob): Promise<*> {
-    return files.setItem(url, blob).then(v => {
-      console.log('successfully saved:', url)
-    }).catch(e => {
-      console.error('error saving:', url, e)
+  'gi.db/files/save': async function (cacheKey: string, blob: Blob): Promise<*> {
+    const keys = await files.getItem('keys') ?? []
+
+    return sbp('okTurtles.eventQueue/queueEvent', 'gi.db/files', () => {
```

Comment:
can you add a comment explaining the reasoning for why an eventQueue is being used?